CONTRIBUTING.md

```diff
@@ -17,3 +17,43 @@ Glint is a family of packages which all live in this repo as a Yarn workspace. T
 - Read the project’s [ARCHITECTURE.md](./ARCHITECTURE.md) to understand the basics of how the code base works.
 
 Once you have made changes and added tests to confirm they work correctly, you can then open a PR and we'll work with you to polish it up and get it landed!
+
```

Comment:
Thanks a ton for adding this! This is all the information i got super stuck on not having when i was first getting in to glint! 

Very needed!